vignettes/creating_linters.Rmd

```diff
@@ -29,51 +30,54 @@ assignment_linter <- function(source_file) {
         linter = "assignment_linter"
         )
     })
+})
 }
 ```
 
-Lets walk through the parts of the linter individually.
+Let us walk through the parts of the linter individually.
```

Comment:
That feels too formal, `Let's` will be fine IMO

Reply:
Feel free to roll back.